sound/soc/sof/intel/hda-codec.c

```diff
@@ -83,7 +83,7 @@ static int hda_codec_probe(struct snd_sof_dev *sdev, int address)
 	return 0;
 #else
 	/* snd_hdac_ext_bus_device_exit will use kfree to free hdev */
-	hdev = kzalloc(sizeof(*hdev), GFP_KERNEL);
+	hdev = devm_kzalloc(sdev->dev, sizeof(*hdev), GFP_KERNEL);
```

Comment:
fix comments, this is no longer true.

Reply:
@plbossart thanks for pointing it out. I will update the patch